README.md

```diff
@@ -147,6 +147,8 @@ lambdaLocal.execute({
 *    `-v, --verboselevel <3/2/1/0>`                     (optional) Default 3. Level 2 dismiss handler() text, level 1 dismiss lambda-local text and level 0 dismiss also the result.
 *    `--envfile <path/to/env/file>`                     (optional) Set extra environment variables from an env file
 *    `--inspect [[host:]port]`                          (optional) Starts lambda-local using the NodeJS inspector (available in nodejs > 8.0.0)
+*    `-W, --watch`                                      (optional) Default false. Starts lambda-local in watch mode.
+*    `--port`                                           (optional) Default 8008. Sets the listening port for watch mode.
```

Comment:
Could you, similarly to `--inspect`, do `--port [port]` with port being optional ?

Reply:
Do you mean also remove the default value here?
https://github.com/ashiina/lambda-local/pull/195/files#diff-cd02b9e0b731a6a1dedefed142f4419bR45

Reply:
I'd rather have a single all-in-one argument for this feature. Instead of `--watch --port 8080` for instance, let's do `--watch 8080`.
You can then default `--watch` with no option to `--watch 8008`, as done in `--inspect`

Reply:
Makes sense. I changed that in my last commit, let me know what do you think